system/SignInWithWeb3.module.scss

```diff
@@ -0,0 +1,8 @@
+.fullWidthButton {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  margin-top: 0.5rem;
```

Comment:
nit: I don't like these margins but its okay.

Reply:
Actually this is unused. Oops.

Reply:
Thanks! Removed.